statusboard/src/components/Footer/Footer.test.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import Footer from './Footer';
+
+describe('Footer: <Footer />', () => {
+  it('should contain a link to open a bug report', function () {
+    const { getByText } = render(<Footer />);
+    expect(getByText('bug report').closest('a')).toHaveAttribute(
+      'href',
+      'https://github.com/codeforamerica/brigade-project-index-statusboard/issues/new?assignees=&labels=&template=bug_report.md&title='
+    );
+  });
+
+  it('should contain a link to open a feature request', function () {
+    const { getByText } = render(<Footer />);
+    expect(getByText('feature request').closest('a')).toHaveAttribute(
+      'href',
+      'https://github.com/codeforamerica/brigade-project-index-statusboard/issues/new?assignees=&labels=&template=feature_request.md&title='
+    );
+  });
```

Comment:
Do you also want to make sure the link to the repository is there?

Reply:
I think I am skipping it, I am not sure that I want the GitHub repository link at all :/ 
I am not sure if it adds anything, or if it is better to have it focused on requesting bug and issue reports